README.md

```diff
@@ -32,15 +32,20 @@ if you need help finding it.
    [RESTful API](https://github.com/Azure/pcs-storage-adapter-dotnet/wiki/API-specs#create-key-value-with-post-id-generated-by-the-service)
    to create a collection.
 
-## Running the service with Visual Studio
-1. Install any edition of [Visual Studio 2017](https://www.visualstudio.com/downloads/). When installing check ".NET Core" workload. 	
+## Running the service with Visual Studio or VS Code
+
+The service can be started from any C# IDE and from the command line.
+The only difference you might notice, is how environment variables
+are configured. See the documentation below for more information.
+
+1. [Install .NET Core 2.x][dotnet-install]
+1. Install any recent edition of Visual Studio (Windows/MacOS) or Visual Studio Code (Windows/MacOS/Linux).
     a. If you already have Visual Studio installed, then ensure you have [.NET Core Tools for Visual Studio 2017](https://www.microsoft.com/net/core#windowsvs2017) installed.
-1.  Open the solution using the pcs-storage-adapter.sln file.
 1. Create your [Azure Cosmos DB account](https://ms.portal.azure.com/#create/Microsoft.DocumentDB) with API type **DocumentDB**
-1. Open the solution in Visual Studio
+1. Open the solution in Visual Studio or VS Code
 1. Either in the project properties Visual Studio or in your system, define the following required environment variables for the both the WebService
```

Comment:
something seems to be missing in the beginning of this sentence

Reply:
What do you suggest?

Reply:
"Either in the project properties Visual Studio or in your system" => "Either in the project properties in Visual Studio or in your system"